src/odbc_connection.cpp

```diff
@@ -579,13 +627,14 @@ set_fetch_size
   // check for that particular SQLSTATE.
   if (return_code == SQL_SUCCESS_WITH_INFO)
   {
+    SQLRETURN   diagnostic_return_code;
     SQLSMALLINT textLength;
     SQLINTEGER  statusRecCount;
     SQLTCHAR    errorSQLState[SQL_SQLSTATE_SIZE + 1];
     SQLINTEGER  nativeError;
     SQLTCHAR    errorMessage[ERROR_MESSAGE_BUFFER_BYTES];
 
-    return_code =
+    diagnostic_return_code =
```

Comment:
```suggestion
    SQLRETURN diagnostic_return_code =
```

Could just define it here.